File: assignment 19.3/task2/task2.java
class task2 {
    public static void is_prime(int n) {
        if (n <= 1) {
            System.out.println(n + " is not a prime number.");
            return;
        }
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                System.out.println(n + " is not a prime number.");
                return;
            }
        }
        System.out.println(n + " is a prime number.");
    }
    public static void main(String[] args) {
        is_prime(17);
        is_prime(20);
    }
}

File: assignment 19.3/task2/task2.py
import math

class Task2:
    @staticmethod
    def is_prime(n):
        if n <= 1:
            print(f"{n} is not a prime number.")
            return
        for i in range(2, int(math.sqrt(n)) + 1):
            if n % i == 0:
                print(f"{n} is not a prime number.")
                return
        print(f"{n} is a prime number.")

if __name__ == "__main__":
    Task2.is_prime(17)
    Task2.is_prime(20)